_topic_map.yml

```diff
@@ -463,10 +471,10 @@ Topics:
     File: using-sriov-multicast
   - Name: Using DPDK and RDMA
     File: using-dpdk-and-rdma
-- Name: OpenShift SDN default CNI network provider
+- Name: OpenShift SDN network provider
```

Comment:
You need to rebase; This definitely should be _default CNI network provider_ now.